scripts/render-r2-import-script.py

```diff
@@ -60,6 +60,12 @@ def render_r2_script(result_document: ResultDocument) -> str:
             sanitized_string = base64.b64encode(b'"FLOSS: %s"' % ss.string.encode("utf-8")).decode("ascii")
             main_commands.append("Ca -0x%x base64:%s @ %d" % (ss.frame_offset, sanitized_string, ss.function))
             ss_len += 1
+    ts_len = 0
```

Comment:
here the lengths aren't used

Reply:
here we are also counting the number of stack strings (ss_len), so I thought this might be something added for future scope.
Should I remove both?

Reply:
yes, please, if needed we could use `len(result_document.strings....)`